File: fix.dart
import 'dart:io';

void main() {
  final file = File('lib/core/models/dish_model.dart');
  var content = file.readAsStringSync();

  final regex = RegExp(r"imageUrl:\s*'https://(images\.unsplash\.com|loremflickr\.com)[^']+'");
  
  content = content.replaceAllMapped(regex, (match) {
    final seed = match.group(0).hashCode.abs() % 1000;
    return "imageUrl: 'https://picsum.photos/seed/$seed/800/800'";
  });

  file.writeAsStringSync(content);
}

File: fix.py
import re

with open('lib/core/models/dish_model.dart', 'r', encoding='utf-8') as f:
    content = f.read()

def replacer(m):
    import hashlib
    seed = int(hashlib.md5(m.group(0).encode('utf-8')).hexdigest(), 16) % 1000
    return "imageUrl: 'https://picsum.photos/seed/" + str(seed) + "/800/800'"

content = re.sub(r"imageUrl:\s*'https://(images\.unsplash\.com|loremflickr\.com)[^']+'", replacer, content)

with open('lib/core/models/dish_model.dart', 'w', encoding='utf-8') as f:
    f.write(content)
